### ISSUE TITLE
AppVeyor: what about migrating from MSBuild to NuGet Automatic Package Restore?

### ISSUE THREAD

**Issue by IlSocio** [restsharp/RestSharp#675]
In order to migrate from MSBuild package restore (which is obsolete) to the new NuGet Automatic Package Restore, some changes are needed on AppVeyor configuration:
http://www.appveyor.com/blog/2014/03/18/about-nuget-package-restore

I already submitted a pull request with those changes in place for the .sln and .prj 
https://github.com/IlSocio/RestSharp/commit/41491e969ead70d260baf4fe47472d7a887dd333
https://github.com/restsharp/RestSharp/pull/674

But the AppVeyour build fails, because AppVeyor currently can't restore packages using the NuGet Automatic Package Restore...

**Comment by issafram** [comment]
I'm surprised this hasn't been changed already.

The solution would be cleaned up alot with Automatic Package Restore.

I think it should be a separate PR though.  Keeping it ATOMIC.

**Comment by hallem** [comment]
Guys thanks for this, I wasn't aware that the MSBuild process was obsolete.  I have a separate AppVeyor build for the PR's so I made the change to the settings to use the before build script of "nuget restore" based on the article you linked to and am re-running the build for that PR.  I do agree that this change should be a separate PR though.  @IlSocio can you make that change once the build is done?

**Comment by IlSocio** [comment]
I'd like to do it, but to be honest, I'm not much familiar with GitHub / PR / AppVeyor, so I didn't understand exactly what I should do at this point and need some clarification...
Should I create two distinct PR? one for the "NuGet Package Restore Migration" and another one with the "Merge Portable to Master branch"?

**Comment by issafram** [comment]
Yes two PRs sounds like a great idea.  If it was me, I'd create two seperate branches from the main fork for my changes if they didn't touch the same files. Then create a PR for each feature/change from each branch.

**Comment by IlSocio** [comment]
I still have to learn how to use github properly, but eventually, I've been able to create the first PR for migrating to the "NuGet Automatic Package Restore" :)
https://github.com/restsharp/RestSharp/pull/6760

**Comment by hallem** [comment]
@IlSocio I'm going to merge in the nuget package restore changes now
